task/04-arrays-tasks.js

```diff
@@ -353,7 +376,7 @@ function getItemsSum(arr) {
  *  [ null, undefined, NaN, false, 0, '' ]  => 6
  */
 function getFalsyValuesCount(arr) {
-   throw new Error('Not implemented');
+   return arr.filter(a => Boolean(a) === false).length;
```

Comment:
`arr.filter(a => !a).length`  it will work

Reply:
I made corrections and wanted to do new pull request but i could't. 
if i do something wrong let me know)